pkg/engine/common.go

```diff
@@ -89,10 +97,43 @@ func currentToLatest(ctx, conn context.Context, m Method, target *Target, tag *[
 		return fmt.Errorf("Failed to get current commit: %v", err)
 	}
 
+	if target.rollback && target.trackBadCommits {
+		if _, ok := BadCommitList[directory]; !ok {
+			BadCommitList[directory] = make(map[string]map[plumbing.Hash]empty)
+		}
+
+		if _, ok := BadCommitList[directory][m.GetKind()]; !ok {
+			BadCommitList[directory][m.GetKind()] = make(map[plumbing.Hash]empty)
+		}
+
+		if _, ok := BadCommitList[directory][m.GetKind()][latest]; ok {
+			klog.Infof("No changes applied to target %s this run, %s currently at %s", directory, m.GetKind(), current)
```

Comment:
update `klog.Infof` to `logger.Infof` - we transitioned to using  zap for logging https://github.com/containers/fetchit/pull/225